theano/printing.py

```diff
@@ -1026,7 +1073,7 @@ def var_name(var):
         if len(varstr) > max_label_size:
             varstr = varstr[:max_label_size - 3] + '...'
         var_str[var] = varstr
-        return varstr
+        return varstr, varlabel
```

Comment:
Pyflakes complains that `varlabel` is undefined. And indeed, I don't see it defined...